feature-libs/organization/administration/components/budget/details/budget-details.component.html

```diff
@@ -72,7 +72,7 @@
     </div>
   </section>
 
-  <section main class="link-list">
+  <section main class="link-list" *ngIf="model.code">
```

Comment:
for better ghost design, I'd probably add the ngIf to the anchor link, so that the section is created before the content is loaded

Reply:
moved to the anchor link